### ISSUE TITLE
Editing markers

### ISSUE THREAD

**Issue by persho** [JanLikar/covid#14]
@JanLikar add a Health status to each marker or we could also add a global setting, but need s to be saved on each marker. "Suspicious", "Confirmed". I think 2 are enough. Use different colours of markers.

Users should be able to edit markers.

Users should be able to delete markers.

**Comment by JanLikar** [comment]
It is already possible to delete them, just click on a marker in add/edit view.

As far as editing goes, I wouldn't want them to be movable, but I feel the notes field should be editable.

**Comment by persho** [comment]
Yes exactly. Also, when we delete them, should we just put a deleted flag and keep them in the db what do you think? @JanLikar

**Comment by JanLikar** [comment]
That makes sense. Creating a ticket now.
